package-lock.json

```diff
@@ -8234,8 +8234,7 @@
           "dependencies": {
             "acorn": {
               "version": "6.1.1",
-              "resolved": "https://registry.npmjs.org/acorn/-/acorn-6.1.1.tgz",
-              "integrity": "sha512-jPTiwtOxaHNaAPg/dmrJ/beuzLRnXtB0kQPQ8JpotKJgTB6rX6c8mlf315941pyjBSaPg8NHXS9fhP4u17DpGA==",
+              "resolved": "",
```

Comment:
On my machine, this change is reverted after running `npm install`. Do you see the same thing ?